lms/djangoapps/certificates/api.py

```diff
@@ -19,6 +19,7 @@
 from opaque_keys.edx.django.models import CourseKeyField
 from organizations.api import get_course_organization_id
 
+from common.djangoapps.course_modes.models import CourseMode
```

Comment:
I know we're trying to move some pieces around without any functional changes. Should we consider doing the work required to avoid the direct import and use of the CourseMode model here?

Reply:
I thought about making a CourseMode dataclass with the constants in it, but that wouldn't necessarily move me forward, so I decided to not add it here. It can definitely be work we do in the future though. Also I consider (maybe wrongly?) that `CourseMode` lives in the same space as `CourseOverview` and friends in that it's the low level thing that other apps are safe to import because it doesn't import from much.